modules/mw.cx.MachineTranslationService.js

```
@@ -158,7 +167,97 @@ mw.cx.MachineTranslationService.prototype.fetchProvidersError = function () {
 	mw.log.error('[CX]', 'Unable to fetch machine translation providers.', arguments);
 };
 
+mw.cx.MachineTranslationService.prototype.cxtoken_error = function () {
```

ESLint

[error] Identifier 'cxtoken_error' is not in camel case.

(camelcase)